ambry-named-mysql/src/main/resources/NamedBlobsSchema.ddl

```diff
@@ -5,7 +5,9 @@ CREATE TABLE IF NOT EXISTS named_blobs (
     blob_id varbinary(50) NOT NULL,
     deleted_ts datetime(6) DEFAULT NULL,
     expires_ts datetime(6) DEFAULT NULL,
-    PRIMARY KEY (account_id, container_id, blob_name)
+    version long(20) NOT NULL,
+    blob_state varchar(20) NOT NULL,
+    PRIMARY KEY (account_id, container_id, blob_name, version)
```

Comment:
what are the steps to adding a new column and change the primary key of a table?
It seems like if we don't prepare for the schema change, we would add version as a new column with NULL value. 

Should we walk out a plan to do schema migration?
Maybe we should do something like this
1. Add version but allow NULL
2. Starting writing version to table
3. Backfill version 
4. Disallow NULL on version
5. Change primary key  to include version.

Reply:
Would this backfill need updating all rows where version is NULL ? If so, we could allow NULL and treat it as T0.

Reply:
Yeah, good point, I raised a ticket to MySQL DBA (https://jira01.corp.linkedin.com:8443/browse/MI-10173), will work with the DBA for the detailed steps of schema migration. 
Since we are adding version to primary key, the version field can't be null

Reply:
Yeah, good point, I raised a ticket to MySQL DBA (https://jira01.corp.linkedin.com:8443/browse/MI-10173), will work with the DBA for the detailed steps of schema migration. 
Since we are adding version to primary key, the version field can't be null